src/components/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+const style = {
```

Comment:
Good job. But it's a best practice to have the styles in a separate file, in this case, it would be `styled.js` and import them here. So, the component file only has the necessary changes.

Reply:
Same comment for the other components.